Makefile

```diff
@@ -123,11 +124,11 @@ clean:
 	rm -rf $(BUILD_DIR)
 .PHONY: integration
 integration: cross
-	go test -ldflags "$(GO_LDFLAGS)" -v -tags integration $(REPOPATH)/integration -timeout 5m -- --remote=true
+	go test -ldflags "$(GO_LDFLAGS)" -v -tags integration $(REPOPATH)/integration -timeout 5m -- --remote=true --gke-cluster-name=$(LOCAL_GKE_CLUSTER_NAME)
 
 .PHONY: integration-local
 integration-local:
-	go test -ldflags "$(GO_LDFLAGS)" -v -tags integration $(REPOPATH)/integration -timeout 5m -remote=false -gac-credentials=$(LOCAL_GAC_CREDENTIALS_PATH)
+	go test -ldflags "$(GO_LDFLAGS)" -v -tags integration $(REPOPATH)/integration -timeout 5m -remote=false -gac-credentials=$(LOCAL_GAC_CREDENTIALS_PATH) --gke-cluster-name=$(LOCAL_GKE_CLUSTER_NAME)
```

Comment:
since remote=false here, I don't think setting the gke-cluster-name will do anything

Reply:
yea forgot about the remote flag being used